README.md

```diff
@@ -435,3 +435,16 @@ unsafe { println!("{}", myUnion.test };
 ```
 
 This is required as the rust compiler can not guarantee the safety of the union, or that the member being addressed exists. The union RFC is [here](https://github.com/tbu-/rust-rfcs/blob/master/text/1444-union.md) and the tracking issue is [here](https://github.com/rust-lang/rust/issues/32836).
+
+### Crate name overrides
+
+`gir` uses simple rule to convert a namespace to a crate name and it sometimes goes wrong. For example, "WebKit2WebExtension" namespace will be converted to "web_kit2_web_extension", which looks bad.
+
+To fix it, the `crate_name_overrides` option can be used.
+
+This also replace FFI crates name.
```

Comment:
The `'` after "crates" is important. Copy and paste it. ;)

```
It also replaces FFI crates' name.
```

Reply:
Done